includes/db.php

```diff
@@ -0,0 +1,17 @@
+<?php
+
+// connect
+$db = new PDO('mysql:host=db; dbname=bikeCollectorApp', 'root', 'password');
+$db->setAttribute(PDO::ATTR_DEFAULT_FETCH_MODE, PDO::FETCH_ASSOC);
```

Comment:
This works just fine, but what about some kind of DB connection function that just returns a PDO instance that can then be passed around the application where needed?